mobile_net/utils/dataset.py

```diff
@@ -0,0 +1,86 @@
+# !/usr/bin/env python
+
+"""
+Loads all images into cache and create a tf.Dataset Object
+"""
+import numpy as np
+import os
+import tensorflow as tf
+import cv2
+import tqdm
+
+
+class Dataset:
+    def __init__(self):
+        # TODO Henning: better pathing
+        self.img_folders = [r"../../both", r"../../no_falcons", r"../../tina", r"../../tom"]
```

Comment:
This should probably be `r"both"` etc. for now since the script will be run from the root of the repository (i.e. the cwd will be the root of the repository)?

If this is turned into a script (see other comments) I will probably add `click`-based argument handling and make it possible to provide a path to the data.